src/api/SFA.DAS.Provider.Events.Api/NLog.config

```diff
@@ -1,45 +1,16 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd"
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-      xsi:schemaLocation="http://www.nlog-project.org/schemas/NLog.xsd NLog.xsd"
-      autoReload="true"
-      throwExceptions="false"
-      internalLogLevel="info" internalLogFile="internal-nlog.txt">
-
+      xmlns:xdt="http://schemas.microsoft.com/XML-Document-Transform">
   <extensions>
     <add assembly="SFA.DAS.NLog.Targets.Redis"/>
   </extensions>
 
-  <!--
-  See https://github.com/nlog/nlog/wiki/Configuration-file
-  for information on customizing logging rules and outputs.
-   -->
   <targets>
-
-    <!--
-    add your targets here
-    See https://github.com/nlog/NLog/wiki/Targets for possible targets.
-    See https://github.com/nlog/NLog/wiki/Layout-Renderers for the possible layout renderers.
-    -->
-
-    <target xsi:type="Redis" name="redis" connectionStringName="Redis" environmentKeyName="EnvironmentName" appName="das-providerevents-api" includeAllProperties="true" layout="${message}">
-      <field name="requestId" layout="${aspnet-traceidentifier}"/>
-      <field name="loggerTitle" layout="${logger}"/>
-    </target>
-
-    <!--
-    Write events to a file with the date in the filename.
-    <target xsi:type="File" name="f" fileName="${basedir}/logs/${shortdate}.log"
-            layout="${longdate} ${uppercase:${level}} ${message}" />
-    -->
+    <target xsi:type="Redis" name="Redis" connectionStringName="LoggingConnectionString" environmentKeyName="EnvironmentName" appName="das-providerevents-api" includeAllProperties="true" layout="${message}"></target>
```

Comment:
App insights Target is missing

<target xsi:type="ApplicationInsights" name="ApplicationInsights" InstrumentationKey="<KeyHere>" environmentKeyName="EnvironmentName" appName="das-providerevents-api" includeAllProperties="true" layout="${message}"></target>

Reply:
As discussed, no app insights target initially, however we do have transaction search visibility from out of the box app insights logging.